### ISSUE TITLE
can we use git-flow in support branch?

### ISSUE THREAD

**Issue by esseti** [petervanderdoes/gitflow-avh#435]
We are using support branches for LTS baiscally.
it would be good to have the same principle and commands for features/hotfix/releases (or just one of these) that starts from the current branch. I guess releases would be the right candidate.

An example
branch: support/3.2.X
git describe: 3.2.1

(i'm in the support/3.2.X branch)
git flow start release 3.2.2
this will start from support/3.2.X 
and will merge to that branch when over. 

Basically it uses the support branch as the master.

Is it feasible? or what should be the strategy to add a new relase in a support branch?

**Comment by cz-dev-ge** [comment]
I am interested in this, too ; )
